### DISCUSSION TITLE
Support for Cockroachdb

### DISCUSSION THREAD

**Discussion by esatterwhite** [Kong/kong#9730] [Help]
Wondering if anyone has tried running kong against cockroach db? does it work? should it work?

**Comment by hbagdi** [comment]
We haven't tested. You may be able to get CRDB working against Koko: https://github.com/kong/koko. The SQL interface that Koko uses is significantly smaller than Kong. An issue on Koko's repository is welcome.

**Reply by esatterwhite** [reply]
I see that koko is archived. Is cockroachdb still a possibility?

**Comment by jrione** [comment]
I've tried to connect Kong (v.3.3) with CockroachDB (v.23.1.5) inside kubernetes cluster, but the results is error: ![image](https://github.com/Kong/kong/assets/57586918/75dffacd-285b-483e-be87-36598e0307df)

I assumed that this error caused by unsupported several PostgreSQL command inside CockroachDB that used by kong.
